Lib/cgi.py

```diff
@@ -577,7 +581,7 @@ def __bool__(self):
             raise TypeError("Cannot be converted to bool.")
         return bool(self.list)
 
-    def read_urlencoded(self):
+    def read_urlencoded(self, separator):
```

Comment:
Why pass in separator? Seems like you can just get the value from self (the other arguments do that too).